### PR TITLE
Gulp bundle ext css

### PR DESCRIPTION
**Pull request by nicolaisueper**
Add bundling of external styles listed in package.json

### CODE REVIEW

**Comment by mischah** [comment]
Updated Dev branch. Please resolve conflicts 😘

**Comment by nicolaisueper** [comment]
Done 😘

File: package.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
     "postmerge": "npm install"
   },
   "bootstrapKickstart": {
-    "bundleCSS": {},
+    "bundleCSS": [],
```
Comment (mischah) [inline package.json]: Please check if bundling works this way.
Test with the following for example:
https://github.com/micromata/bootstrap-kickstart/commit/015b25d99d03b61a572c7de26c6a6de5f61cc7f4#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L78

Comment (nicolaisueper) [inline package.json]: Now it works :)

**Comment by mischah** [comment]
Closed in favor of #144

**Pull request closed without merging**